src/agents/memory-optimizer/memory-optimization-agent.ts

```
@@ -0,0 +1,857 @@
+
+      // Build prompts for LLM analysis (with optional MCP sampling)
+      const systemPrompt = this.config.enableMCPSampling
+        ? await this.promptManager.buildContextAwareSystemPrompt(
+            'analyzer',
+            strategy,
+            mcpContext,
+            clientProjectRoot,
+            repository,
+            branch,
+            true,
+            this.config.samplingStrategy || 'representative'
```

ESLint

[error] Insert `,`

(prettier/prettier)